### PR TITLE
 Make admin user creation configurable

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -11,13 +11,19 @@ if test $# -eq 0; then
     exit 1
 fi
 
-fabmanager list-users --app superset # Run this to report any config errors visibly. https://github.com/dpgaspar/Flask-AppBuilder/issues/742
-user_count="$(fabmanager list-users --app superset | sed -n '/^username/p' | wc -l)"
-if test "${user_count}" = 0; then
-    stderr "No users exist. Creating admin user."
-    fabmanager create-admin --app superset --username admin --password admin \
-        --firstname Admin --lastname Admin --email admin@localhost
+SUPERSET_ADMIN_USERNAME="${SUPERSET_ADMIN_USERNAME-admin}"
```
Comment (sopel39) [inline docker-entrypoint.sh:14]: bug, rewrite to: `${SUPERSET_ADMIN_USERNAME:-admin}`

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -11,13 +11,19 @@ if test $# -eq 0; then
...
+SUPERSET_ADMIN_PASSWORD="${SUPERSET_ADMIN_PASSWORD:-admin}"
+if test ! -z "${SUPERSET_ADMIN_USERNAME}"; then
```
Comment (sopel39) [inline docker-entrypoint.sh:16]: Can `SUPERSET_ADMIN_USERNAME` be unset? Can it be empty? Maybe fail if it is empty.

**Review by sopel39 — APPROVED** [review]
LGTM

File: docker-entrypoint.sh (hunk 1 above)
Comment (findepi) [reply docker-entrypoint.sh:14]: lack of `:` is intentional. https://i.stack.imgur.com/T2Fp8.png

File: docker-entrypoint.sh (hunk 2 above)
Comment (findepi) [reply docker-entrypoint.sh:16]: I consciously allow `SUPERSET_ADMIN_USERNAME` be empty. In that case, no user is created.

**Pull request merged**